ros2nodl/ros2nodl/_verb/_validate.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2020 Canonical Ltd.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License Version 3 as published by
+# the Free Software Foundation.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+from pathlib import Path
+import pprint
+import shutil
+import sys
+
+from argcomplete.completers import FilesCompleter
+import nodl
+from ros2cli.verb import VerbExtension
+
+
+class _ValidateVerb(VerbExtension):
+    """Validate NoDL XML documents."""
+
+    def add_arguments(self, parser: argparse.ArgumentParser, cli_name: None = None):
+        command_group = parser.add_mutually_exclusive_group(required=True)
+        # Ignoring type because of https://github.com/python/typeshed/issues/1878
+        command_group.add_argument(  # type: ignore
+            'file', nargs='*', default=[], help='Specific .nodl.xml file(s) to validate.'
+        ).completer = FilesCompleter(allowednames=['nodl.xml'], directories=False)
+        command_group.add_argument(
+            '-a',
+            '--all',
+            action='store_true',
+            help='Validate all .nodl.xml files in the current directory.',
```

Comment:
Similar to above, I'm not sure this arg is necessary. Thoughts?

Reply:
Fixed in 1b16352